server/arweave/index.ts

```diff
@@ -177,10 +175,12 @@ export async function convertARPriceToLIKE(ar: ArweavePrice, {
 export async function convertARPricesToLIKE(ar: ArweavePrice,
   { margin = 0.05, decimal = 0 } = {},
 ): Promise<ArweavePriceWithLIKE> {
+  const { data } = await axios.get(COINGECKO_PRICE_API);
+  const { likecoin, arweave: arweavePrice } = data;
   if (!(ar.list && ar.list.length)) {
-    return convertARPriceToLIKE(ar, { margin, decimal });
+    return convertARPriceToLIKE(ar, { margin, decimal }, likecoin, arweavePrice);
```

Comment:
instead of adding `likecoin`, can we calculate `priceRatio` as input for `convertARPriceToLIKE` and keep the original function interface instead?